packages/manager/modules/telecom-spare/src/index.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Difference between loading a module and importing a module
```

Comment:
I suppose these comments are more generic knowledge and should probably in an internal documentation rather than in the module documentation :)